FirebaseRemoteConfig/Sources/RCNConfigRealtime.m

```diff
@@ -53,6 +53,12 @@
 NSTimeInterval timeoutSeconds = 4320;
 NSInteger FETCH_ATTEMPTS = 3;
 
+// Retry parameters
+NSInteger MAX_RETRY = 7;
+NSInteger MAX_RETRY_COUNT = 7;
+NSInteger RETRY_SECONDS = 3;
```

Comment:
Why is this initialized here but then set below?

Reply:
gMaxRetryCount keeps track of retries remaining and gMaxRetries is the initial count that gMaxRetryCount will be restarted to. Changed gRetrySeconds to being initialized in the constructor.